src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -750,12 +750,6 @@ because xbuild doesn't support framework reference assemblies.
 		/>
 	</CreateProperty>
 
-	<CreateProperty Value="$(_AndroidToolsDirectory)bin\" Condition="Exists ('$(_AndroidToolsDirectory)bin\')">
-		<Output TaskParameter="Value" PropertyName="LintToolPath"
```

Comment:
This and the commit message confuses me.

We're going to make `$(LintToolPath)` unset/empty by default, and add additional `File.Exists()` checking to the `<Lint/>` task...?

Why not instead update the `<ResolveSdks/>` Task to output a `ResolveSdks.LintToolPath` property, and let `<ResolveSdks/>` check for the appropriate directories?